src/styles/imageEditor.scss

```diff
@@ -0,0 +1,51 @@
+.whppt-image-editor {
+  @apply flex flex-col gap-4;
+
+  &__device-picker {
+    @apply flex gap-12 text-whpptLightMode;
+  }
+
+  &__device-select {
+    @apply cursor-pointer;
+
+    &--active {
+      @apply text-whpptActive underline;
+    }
+  }
+
+  &__gallery-actions {
+    @apply flex justify-between;
+
+    &__button {
+      @apply text-whpptLightMode cursor-pointer;
```

Comment:
seeing curser-pointer is an anti-pattern. If the element needs to be a link or a button make it a button or a link. That will apply the cursor pointer. Then style the element to look correct.